arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/TxInitializationSection.java

```diff
@@ -38,73 +38,66 @@
 import org.hyperledger.besu.evm.worldstate.WorldView;
 
 public class TxInitializationSection extends TraceSection implements PostTransactionDefer {
+
+  @Getter private final int hubStamp;
+  final AccountFragment.AccountFragmentFactory accountFragmentFactory;
+
   @Getter private final AccountSnapshot senderGasPayment;
   @Getter private final AccountSnapshot senderGasPaymentNew;
 
-  @Getter private final AccountSnapshot senderValueTransfer; // = senderGasPaymentNew
+  @Getter private final AccountSnapshot senderValueTransfer;
   @Getter private final AccountSnapshot senderValueTransferNew;
 
   @Getter private final AccountSnapshot recipientValueReception;
   @Getter private final AccountSnapshot recipientValueReceptionNew;
 
-  @Getter private final AccountSnapshot senderUndoingValueTransfer; // = senderValueTransferNew
+  @Getter private AccountSnapshot senderUndoingValueTransfer;
   @Getter private AccountSnapshot senderUndoingValueTransferNew;
 
-  @Getter private final AccountSnapshot recipientUndoingValueReception;
-  // = recipientValueReceptionNew
+  @Getter private AccountSnapshot recipientUndoingValueReception;
   @Getter private AccountSnapshot recipientUndoingValueReceptionNew;
 
-  final AccountFragment.AccountFragmentFactory accountFragmentFactory;
-  final Wei value;
-  final DomSubStampsSubFragment senderDomSubStamps;
-  final DomSubStampsSubFragment recipientDomSubStamps;
+  @Getter private final ContextFragment initializationContextFragment;
 
   public TxInitializationSection(Hub hub, WorldView world) {
-    super(hub, (short) 5);
-
-    // This ensures resolvePostTransaction is executed
+    super(hub, (short) 8);
     hub.defers().scheduleForPostTransaction(this);
 
+    hubStamp = hub.stamp();
```

Comment:
Required for the `DOM`/`SUB` stamps in case of a `REVERT`ed transaction. The `HUB_STAMP` that you use is the one of the start of the transaction (in the `SUB_STAMP`.)